rest/route/service.go

```diff
@@ -135,8 +136,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	// client_urls is used by the agent monitor deploy job which does not pass in user info
 	app.AddRoute("/distros/{distro_id}/client_urls").Version(2).Get().RouteHandler(makeGetDistroClientURLs(env))
 
-	// Middleware is omitted for webhook routes because they cannot be called by users and validation occurs in Parse function.
```

Comment:
Rather than remove it, ieems like this comment should be moved down to L140 since it still applies to the AWS webhooks.